client/extensions/woocommerce/app/settings/shipping/shipping-package-list.js

```diff
@@ -41,7 +41,7 @@ class ShippingPackageList extends Component {
 			<div>
 				<ExtendedHeader
 					label={ translate( 'Packages' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Theme Packages' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).